README.rst

```diff
@@ -27,20 +27,28 @@ Example Usage
 
 .. code-block:: python
 
-   from adb_shell.adb_device import AdbDeviceTcp
+   from adb_shell.adb_device import AdbDeviceTcp, AdbDeviceUsb
    from adb_shell.auth.sign_pythonrsa import PythonRSASigner
 
    # Connect (no authentication necessary)
    device1 = AdbDeviceTcp('192.168.0.111', 5555, default_timeout_s=9.)
    device1.connect(auth_timeout_s=0.1)
 
    # Connect (authentication required)
-   with open('path/to/adbkey') as f:
+   with open('path/to/adbkey', 'rb') as f:
        priv = f.read()
    signer = PythonRSASigner('', priv)
    device2 = AdbDeviceTcp('192.168.0.222', 5555, default_timeout_s=9.)
    device2.connect(rsa_keys=[signer], auth_timeout_s=0.1)
 
+   # Connect via USB (requires adb-shell[usb])
+   with open('path/to/adbkey', 'rb') as f:
+       priv = f.read()
+   signer = PythonRSASigner('', priv)
+   device3 = AdbDeviceUsb('ab78c6ef')
```

Comment:
It would be nice to have a cleaner way to instantiate the USB handle. I think that’s possible? But you know more about how the USB code works than I do.

Reply:
I think it depends on what you mean by 'nice'.. Here's a short review of how things work.
- The original `adb` uses the same serial number string.
- If no parameter is provided, the default behavior is to find exactly one ADB USB device, and error out otherwise.
- The `python-adb` package follows the above, but also allows you to provide a USB port path (you could think of it as a USB-address) instead of a serial number. (`PortPathMatcher`)
- I'll have to study more about USB in order to come up with other ways.

What we could do right away is implement the default behavior, which would be nice, indeed.

Reply:
> * The original `adb` uses the same serial number string to identify a specific device.
> * If no parameter is provided, the default behavior is to find exactly one ADB USB device, and error out otherwise.
> * The `python-adb` package follows the above, but also allows you to provide a USB port path (you could think of it as a USB-address) instead of a serial number. (`PortPathMatcher`)

Thanks for the explanation. So the first two of these are implemented? That sounds pretty good to me.

Reply:
They are now :)
Well, almost. The default behavior is to get the **first** ADB device. 
There may be others, which means the result might not be deterministic (or even predictable).
I'll probably want to change this very soon.